<stream>:1:1: Unexpected token

" , "internal" : true , "results" : [] }
Internal error
Library file not found: why3.Foo__lists__listDefault_Initial_Condition___object.
None
called process error: {"error" : "File /"gnatprove/foo/../foo.mlw/", line 5672, characters 1-62:
gnatprove: error during flow analysis and proof
